.github/workflows/create-board.yaml

```diff
@@ -0,0 +1,50 @@
+name: Create Board
+
+on:
+  workflow_call
+
+jobs:
+  create-board:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.MY_REPO_PAT }}
+          submodules: recursive
+
+      - name: Install kicad
+        run: |
+          sudo add-apt-repository --yes ppa:kicad/kicad-7.0-releases
+          sudo apt-get install kicad
+          pip install kicad-netlist-reader
+
+      - name: Generate Gerber Files
+        run: |
+          bash hellen-one/kicad/bin/export.sh hellen88bmw
```

Comment:
now we really want https://github.com/andreika-git/hellen-one/issues/308 :)

Reply:
https://github.com/andreika-git/hellen-one/pull/310